### PR TITLE
Adds more.../ less... link where the content of a field is too long

### LINKED ISSUE #413: Hide long values behind "More..." link
Truncate long values with a link to reveal the rest of the value.  This will make metadata display better by not having long values hide other fields below.

### PR DESCRIPTION
**Pull request by christinach**
closes #413

### CODE REVIEW

**Comment by christinach** [comment]
https://dpul-staging.princeton.edu/cotsen/catalog/5712m9875#?c=0&m=0&s=0&cv=0&xywh=-10245%2C-402%2C24996%2C8000 . (testing with a 300 character limit)

**Comment by escowles** [comment]
@christinach 👍 Looks good in staging!

**Comment by christinach** [comment]
@tpendragon I have a difficulty adding a spec to view the document through the spec in the UI. I added one spec on the catalog controller only to see that the field with the long characters is there; and one spec as a feature to check the more link on the description field. The part which will fail the feature spec is commented.

**Comment by christinach** [comment]
@jrgriffiniii created this spec https://github.com/pulibrary/pomegranate/compare/issues-484-jrgriffiniii-catalog-spec
I'm able to return the page body but still not able to find the css classes which are added through javascript.

**Comment by christinach** [comment]
Locally an update in the spec was the string `"panoramic"*50` because the character limit I have set in the JS is 300.

File: spec/controllers/omniauth_callbacks_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Users::OmniauthCallbacksController do
+  before { request.env['devise.mapping'] = Devise.mappings[:user] }
+
+  # describe 'logging in' do
```
Comment (tpendragon) [inline spec/controllers/omniauth_callbacks_controller_spec.rb]: Can you delete this commented code?

Comment (christinach) [reply spec/controllers/omniauth_callbacks_controller_spec.rb]: thank you, I did.

**Pull request merged**